tests/functional/firefox/test_mobile.py

```diff
@@ -8,61 +8,121 @@
 from pages.firefox.mobile import FirefoxMobilePage
 
 
+# mobile - send to device
```

Comment:
It's been a while since this branch was resurrected - I'd suggest pushing this to `run-integration-tests` to make sure these still pass ok.